Add primary keys to tables that are missing them

Jelle-S opened the pull request:
# Description

Adds primary keys to tables where they are missing.
See See laravel/framework#15770 for a thorough explanation and more context. 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested on our environment where the migrations table was missing the `id` column (shows you we've been using this since at least 2016 :sweat_smile: ) and with the default database you get when installing snipe it


# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

Jelle-S committed:
Add primary key to custom_field_custom_fieldset

Some mysql clusters (like Percona-XtraDB-Cluster) prohibit the use of DML
commands on a table. It's also just a good practice to add it.

Jelle-S committed:
Add primary key to migrations table if it doesn't exist yet

Some mysql clusters (like Percona-XtraDB-Cluster) prohibit the use of DML
commands on a table. It's also just a good practice to add it.

See https://github.com/laravel/framework/pull/15770

uberbrady reviewed (changes requested):
If you can retarget these changes to the Develop branch, we'll take them.

snipe commented:
I’m still kind of unclear what problem this solves, exactly?

Jelle-S committed:
Add primary key to the password_resets table

Some mysql clusters (like Percona-XtraDB-Cluster) prohibit the use of DML
commands on a table. It's also just a good practice to add it.

Jelle-S commented:
> I’m still kind of unclear what problem this solves, exactly?

Some database systems (like XtraDB cluster) require explicit primary keys for _all_ tables, so this PR adds them to those that are missing them.

I'm closing this one and created a new one for develop: https://github.com/snipe/snipe-it/pull/9363
